src/Components/Login/Login.tsx

```diff
@@ -26,10 +26,18 @@ export const Login = ({ setLoggedIn }: Props) => {
                 e.preventDefault();
 
                 setUserInfo({ email: email, password: password });
+                console.log(
+                    userInfo,
+                    "after set in log in for email and password",
+                    email,
+                    password
```

Comment:
the userInfo is not updated until the second input click from the user. Then the login functionality works as intended. The state has the information when the userInfo is called but when the if() on line 37 happens, the userInfo is not yet updated with the setUserInfo call. spent 30~ min debugging, then decided to table this for a morning stand-up roadblock.